packages/babel-helper-define-map/src/index.js

```diff
@@ -98,7 +99,7 @@ export function push(
 }
 
 export function hasComputed(mutatorMap: Object): boolean {
-  for (const key in mutatorMap) {
+  for (const key in own(mutatorMap)) {
```

Comment:
Do we need the macro? It's pretty easy to do `for (const key of Object.keys(mutatorMap)) {`.

Reply:
I already had the macro in another project so I published it and added there, but I can remove it if you prefer.
My fear is that `for ... of + Object.keys` is slower than `for ... in + Object.hasOwnProperty`, and it is used in some perf-sensitive places in `@babel/traverse`.

Reply:
Nvm, I was wrong https://jsperf.com/for-in-vs-for-ok/1